libraries/AP_Baro/AP_Baro_Backend.cpp

```diff
@@ -56,3 +59,28 @@ void AP_Baro_Backend::_copy_to_frontend(uint8_t instance, float pressure, float
     _frontend.sensors[instance].temperature = temperature;
     _frontend.sensors[instance].last_update_ms = now;
 }
+
+#define FILTER_KOEF 0.1
+
+bool AP_Baro_Backend::pressure_ok(float press) {
+    bool ret=true;
+    
+    if(isinf(press) || isnan(press)) return false;
+    
+    if(is_zero(_mean_pressure)){
+        _mean_pressure = press;
+    } else {
+        float range = _frontend.get_filtrer_range();
+        float d = abs(_mean_pressure-press)/(_mean_pressure+press);
+        float k = FILTER_KOEF;
+
+        if(!is_zero(range) && d*200 > range) { // check the difference from mean value outside allowed range
```

Comment:
I am sorry but I didn't understand where the 200 come from . Can you explain a bit ?

Reply:
(a-b)/((a+b) / 2) should be, then to get percent we should to multiply by 100. But 2 in formula from divider just moved out to reduce calculations so to get percents it will be (2*100)

Reply:
then why divide the normalised error by 2 ? (sorry, I forget basic math ... I just try to understand)

Reply:
difference should be divided by mean, not by sum

Reply:
haaaaa ok it makes seens now ! understood ! thanks !